.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Test PR
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This shouldn't be here. Merges to `main` are prohibited and ALL changes happen through PRs.